### DISCUSSION TITLE
Education license is activated but co-pilot does not work.

### DISCUSSION THREAD

**Discussion by fatrybl** [community/community#139476] [GitHub Education]
### Hi everyone,

Im asking here cause cant wait more (3 weeks already). I created ticket #2989690, an issue in [co-pilot specific area](https://github.com/orgs/community/discussions/139174) at all. i cant use my education pack. Maybe someone here could help me ? everything is described in details [here](https://github.com/orgs/community/discussions/139174)

**Comment by Chanduporalla** [comment]
Hi! I understand your frustration with the delay; waiting three weeks can be really challenging. I’ve checked ticket #2989690, and I see the details you've provided about the issue with the education pack. While I may not have direct access to resolve it, I recommend reaching out to support through other channels, like social media or their official forums, as well. Sometimes, that can expedite the process. I hope you get it sorted out soon!

**Reply by fatrybl** [reply]
Hello
Thank you for the reply. I will try to follow the recommendation of @davevad93 and create a ticket there. Well, I`ve already created a ticket here, in Co-Pilot forum and a personal ticket.  Seems like its not directly related to my problem so i will try not to spam my issue everywhere but you are right, social media could help maybe to speed up the process

**Comment by davevad93** [comment]
Hi @fatrybl ,

Are you experiencing this problem with just `Copilot` or with everything offered as part of the `Student Developer Pack`? Try going to the [pack offers page](https://education.github.com/pack), scroll down to the "All offers" section and see if you can redeem any of the available offers. Another question, have you also raised a ticket [here](https://support.github.com/contact/education), selecting `I am a verified student having trouble redeeming my Global Campus offers`? That's the correct category for this kind of issues. If you have already opened the ticket there, you need to be patient because support can take a while to respond, I hope they can help you resolve it as soon as possible.

**Reply by fatrybl** [reply]
Hi! thank you for the reply. 
1) I don`t use any other "paid" offers. But i do use PyCharm and co-pilot can be downloaded and updated there as well as in VS code. Additionally i just signed-up in Notion and got my account associated with Github Students pack. So yes, it works and i can use the AI-chat in Notion. 
2) No, i haven`t tried it. But definitely will do it. 
Thank you so much !

**Reply by davevad93** [reply]
You're welcome @fatrybl , hope it gets fixed asap! 👍
